### PR TITLE
Add details view

### PR DESCRIPTION
**Pull request by pgetta**

### CODE REVIEW

File: apps/search-poc/src/views/DetailsContainer.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import useEntity from '../hooks/useEntity';
+import Details from '../components/Details';
+
+const DetailsContainer: React.FunctionComponent<{
+  params: {
+    selfUrl: string;
+  };
+}> = props => {
+  const { entity, loading, error } = useEntity(props.params.selfUrl);
```
Comment (julienmachon) [inline apps/search-poc/src/views/DetailsContainer.ts]: Not sure this hook is necessary... you can directly do:
```suggestion
  const { data, loading, error } = useNexus(nexus => nexus.httpGet({ path: props.params.selfUrl }))
```

File: apps/search-poc/src/hooks/useEntity.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
```
Comment (julienmachon) [inline apps/search-poc/src/hooks/useEntity.ts]: As I said in another comment, I think this hook is not necessary. All this code can be replaced with:
```tsx
const { data, loading, error } = useNexus(nexus => nexus.httpGet({ path: decodeURIComponent(props.params.selfUrl) }))
```

File: apps/search-poc/src/index.tsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ import { NexusProvider } from '@bbp/react-nexus';
 import { setUpSession, setToken } from './utils/auth';
 import Header from './components/Header';
 import SparqlView from './views/Sparql';
-import DetailsView from './views/Details';
+import DetailsContainerView from './views/DetailsContainer';
```
Comment (julienmachon) [inline apps/search-poc/src/index.tsx]: Not sure I get why this file has been renamed to `DetailsContainer`... We want to create a new view called `Details View` so why not simply have a `Details` component in the `View` folder?

**Comment by pgetta** [comment]
@julienmachon I've refactored it to follow view/container/component approach.

File: apps/search-poc/src/containers/Details.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import { useNexus } from '@bbp/react-nexus';
+
+import DetailsComponent from '../components/Details';
+
+
+const DetailsContainer: React.FunctionComponent<{
+  selfUrl: string;
+}> = props => {
+  const { data, loading } = useNexus(nexus => nexus.httpGet({ path: props.selfUrl }));
+
```
Comment (julienmachon) [inline apps/search-poc/src/containers/Details.tsx]: I think I'd add a 
```suggestion
if (error) {
return <p>something went wrong...</p>; // or whatever
}
```

File: apps/search-poc/src/components/Details.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import { Spin } from 'antd';
+
+
+const DetailsComponent: React.FunctionComponent<{
+  data: any;
```
Comment (julienmachon) [inline apps/search-poc/src/components/Details.tsx]: `any` is bad! :wink: 

Also I'd prefer to see something more explicit like:
```suggestion
  id: string;
  name: string;
  description: string;
  brainRegion: {
    id: string;
    label: string;
  };
```

File: apps/search-poc/src/containers/Details.tsx (hunk 4 above)
Comment (pgetta) [reply apps/search-poc/src/containers/Details.tsx]: Error handling has been added

File: apps/search-poc/src/components/Details.tsx (hunk 5 above)
Comment (pgetta) [reply apps/search-poc/src/components/Details.tsx]: I've also added types for common resources

**Pull request merged**